starwars/src/components/Characters.js

```diff
@@ -3,17 +3,28 @@ import React from "react";
 const Character = props => {
     return (
         <div className="card">
-            <div className="card_header">{props.char.name}</div>
+            <h2 className="card_header">{props.char.name}</h2>
             
             <div className="card_content">
-                <h2 className="charDOB">Date of birth: {props.char.birth_year}</h2>
-                <h2 className="charSex">{props.char.gender}</h2>
+                <h3 className="charDOB">Date of birth: {props.char.birth_year}</h3>
+                <h3 className="charSex">{props.char.gender}</h3>
 
-                    <div className="card-content_extra">
-                        Eye color: {props.char.eye_color}
-                        
-                    </div>
+                <div className="card-content_info">
```

Comment:
your prop passing is strong and your styles are going ok. but your commit messages are a little light